Lmod has detected the following error: The following module(s) are unknown:
"system_env"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore-cache load "system_env"

Also make sure that all modulefiles written in TCL start with the string
#%Module



pplacer/1.1.alpha19 : Several complimentary python scripts are available, do 'module help pplacer/1.1.alpha19' for more information
To set up a CheckM data directory, see 'module help CheckM/1.1.3'
It seems that the CheckM data folder has not been set yet or has been removed. Please run 'checkm data setRoot'.

Path [/home/anniei/.checkm] does not exist so I will attempt to create it
Path [/home/anniei/.checkm] has been created and you have permission to write to this folder.
(re) creating manifest file (please be patient).
[2025-04-26 21:28:51] INFO: CheckM v1.1.3
[2025-04-26 21:28:51] INFO: checkm lineage_wf -x fa --threads 8 /home/anniei/Genome_analysis/analyses/DNA/04_binning/metabat_SRR4342129 /home/anniei/Genome_analysis/analyses/DNA/05_quality/checkm_SRR4342129
[2025-04-26 21:28:51] INFO: [CheckM - tree] Placing bins in reference genome tree.
[2025-04-26 21:28:51] INFO: Identifying marker genes in 24 bins with 8 threads:
    Finished processing 0 of 24 (0.00%) bins.Process Process-6:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-8:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-2:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-3:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-4:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-5:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-7:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-9:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'

[2025-04-26 21:29:14] INFO: Saving HMM info to file.
[2025-04-26 21:29:14] INFO: Calculating genome statistics for 24 bins with 8 threads:
    Finished processing 0 of 24 (0.00%) bins.    Finished processing 1 of 24 (4.17%) bins.    Finished processing 2 of 24 (8.33%) bins.    Finished processing 3 of 24 (12.50%) bins.    Finished processing 4 of 24 (16.67%) bins.    Finished processing 5 of 24 (20.83%) bins.    Finished processing 6 of 24 (25.00%) bins.    Finished processing 7 of 24 (29.17%) bins.    Finished processing 8 of 24 (33.33%) bins.    Finished processing 9 of 24 (37.50%) bins.    Finished processing 10 of 24 (41.67%) bins.    Finished processing 11 of 24 (45.83%) bins.    Finished processing 12 of 24 (50.00%) bins.    Finished processing 13 of 24 (54.17%) bins.    Finished processing 14 of 24 (58.33%) bins.    Finished processing 15 of 24 (62.50%) bins.    Finished processing 16 of 24 (66.67%) bins.    Finished processing 17 of 24 (70.83%) bins.    Finished processing 18 of 24 (75.00%) bins.    Finished processing 19 of 24 (79.17%) bins.    Finished processing 20 of 24 (83.33%) bins.    Finished processing 21 of 24 (87.50%) bins.    Finished processing 22 of 24 (91.67%) bins.    Finished processing 23 of 24 (95.83%) bins.    Finished processing 24 of 24 (100.00%) bins.
[2025-04-26 21:29:15] INFO: Extracting marker genes to align.
[2025-04-26 21:29:15] ERROR: Models must be parsed before identifying HMM hits.

  Controlled exit resulting from an unrecoverable error or warning.
[2025-04-26 21:29:16] INFO: CheckM v1.1.3
[2025-04-26 21:29:16] INFO: checkm lineage_wf -x fa --threads 8 /home/anniei/Genome_analysis/analyses/DNA/04_binning/metabat_SRR4342133 /home/anniei/Genome_analysis/analyses/DNA/05_quality/checkm_SRR4342133
[2025-04-26 21:29:16] INFO: [CheckM - tree] Placing bins in reference genome tree.
[2025-04-26 21:29:17] INFO: Identifying marker genes in 29 bins with 8 threads:
    Finished processing 0 of 29 (0.00%) bins.Process Process-2:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-7:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-6:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-8:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-5:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-3:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-9:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'
Process Process-4:
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerGeneFinder.py", line 122, in __processBin
    hmmModelFile = markerSetParser.createHmmModelFile(binId, markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 330, in createHmmModelFile
    markerFileType = self.markerFileType(markerFile)
  File "/usr/local/lib/python3.7/site-packages/checkm/markerSets.py", line 430, in markerFileType
    with open(markerFile, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/anniei/.checkm/hmms/phylo.hmm'

[2025-04-26 21:29:32] INFO: Saving HMM info to file.
[2025-04-26 21:29:32] INFO: Calculating genome statistics for 29 bins with 8 threads:
    Finished processing 0 of 29 (0.00%) bins.    Finished processing 1 of 29 (3.45%) bins.    Finished processing 2 of 29 (6.90%) bins.    Finished processing 3 of 29 (10.34%) bins.    Finished processing 4 of 29 (13.79%) bins.    Finished processing 5 of 29 (17.24%) bins.    Finished processing 6 of 29 (20.69%) bins.    Finished processing 7 of 29 (24.14%) bins.    Finished processing 8 of 29 (27.59%) bins.    Finished processing 9 of 29 (31.03%) bins.    Finished processing 10 of 29 (34.48%) bins.    Finished processing 11 of 29 (37.93%) bins.    Finished processing 12 of 29 (41.38%) bins.    Finished processing 13 of 29 (44.83%) bins.    Finished processing 14 of 29 (48.28%) bins.    Finished processing 15 of 29 (51.72%) bins.    Finished processing 16 of 29 (55.17%) bins.    Finished processing 17 of 29 (58.62%) bins.    Finished processing 18 of 29 (62.07%) bins.    Finished processing 19 of 29 (65.52%) bins.    Finished processing 20 of 29 (68.97%) bins.    Finished processing 21 of 29 (72.41%) bins.    Finished processing 22 of 29 (75.86%) bins.    Finished processing 23 of 29 (79.31%) bins.    Finished processing 24 of 29 (82.76%) bins.    Finished processing 25 of 29 (86.21%) bins.    Finished processing 26 of 29 (89.66%) bins.    Finished processing 27 of 29 (93.10%) bins.    Finished processing 28 of 29 (96.55%) bins.    Finished processing 29 of 29 (100.00%) bins.
[2025-04-26 21:29:33] INFO: Extracting marker genes to align.
[2025-04-26 21:29:33] ERROR: Models must be parsed before identifying HMM hits.

  Controlled exit resulting from an unrecoverable error or warning.
